src/app/blogs/blog/page.tsx

```diff
@@ -60,11 +60,13 @@ export default function OneBlog({ searchParams }: MetadataProps) {
         imageUrl={blog.imageUrl}
         bodyText={blog.bodyText}
       />{' '}
-      <DisplayPodcast
-        scriptSrc={blog.scriptSrc}
-        containerId={blog.containerId}
-        singleBlog={true}
-      />
+      {blog.scriptSrc != '' && blog.containerId != '' && (
+        <DisplayPodcast
+          scriptSrc={blog.scriptSrc}
+          containerId={blog.containerId}
```

Comment:
Not the most important thing, but try to destructure when a property of a JS object is used multiple times - for better readability, performance & maintainability.  

For instance, we could've done ` const { scriptSrc, containerId } = blog ` - something like that!